src/pkg/count/keys.go

```diff
@@ -1,42 +1,62 @@
 package count
 
-type key string
+type Key string
 
 const (
 	// count of bucket-tokens consumed by api calls.
-	APICallTokensConsumed key = "api-call-tokens-consumed"
+	APICallTokensConsumed Key = "api-call-tokens-consumed"
 	// count of api calls that resulted in failure due to throttling.
-	ThrottledAPICalls key = "throttled-api-calls"
+	ThrottledAPICalls Key = "throttled-api-calls"
 )
 
 // Tracked during backup
 const (
 	// amounts reported by kopia
-	PersistedCachedFiles          key = "persisted-cached-files"
-	PersistedDirectories          key = "persisted-directories"
-	PersistedFiles                key = "persisted-files"
-	PersistedHashedBytes          key = "persisted-hashed-bytes"
-	PersistedNonCachedFiles       key = "persisted-non-cached-files"
-	PersistedNonMetaFiles         key = "persisted-non-meta-files"
-	PersistedNonMetaUploadedBytes key = "persisted-non-meta-uploaded-bytes"
-	PersistedUploadedBytes        key = "persisted-uploaded-bytes"
-	PersistenceErrors             key = "persistence-errors"
-	PersistenceExpectedErrors     key = "persistence-expected-errors"
-	PersistenceIgnoredErrors      key = "persistence-ignored-errors"
+	PersistedCachedFiles          Key = "persisted-cached-files"
+	PersistedDirectories          Key = "persisted-directories"
+	PersistedFiles                Key = "persisted-files"
+	PersistedHashedBytes          Key = "persisted-hashed-bytes"
+	PersistedNonCachedFiles       Key = "persisted-non-cached-files"
+	PersistedNonMetaFiles         Key = "persisted-non-meta-files"
+	PersistedNonMetaUploadedBytes Key = "persisted-non-meta-uploaded-bytes"
+	PersistedUploadedBytes        Key = "persisted-uploaded-bytes"
+	PersistenceErrors             Key = "persistence-errors"
+	PersistenceExpectedErrors     Key = "persistence-expected-errors"
+	PersistenceIgnoredErrors      Key = "persistence-ignored-errors"
+	// amounts reported by producers
+	CollectionMoved      Key = "collection-moved"
+	CollectionNew        Key = "collection-state-new"
+	CollectionNotMoved   Key = "collection-state-not-moved"
+	CollectionTombstoned Key = "collection-state-tombstoned"
+	Collections          Key = "collections"
+	ItemsAdded           Key = "items-added"
+	ItemsRemoved         Key = "items-removed"
+	MissingDelta         Key = "missing-delta-token"
+	NewDeltas            Key = "new-delta-tokens"
+	NewPrevPaths         Key = "new-previous-paths"
+	NoDeltaQueries       Key = "cannot-make-delta-queries"
+	PrevDeltas           Key = "previous-deltas"
 	// amounts reported by data providers
-	ProviderItemsRead key = "provider-items-read"
+	ProviderItemsRead Key = "provider-items-read"
```

Comment:
Could you please add a description for some of these?

Reply:
I've added a few.  Kinda difficult from my perspective as implementor to know what makes sense and what doesn't.  If you have additional pointers to which ones are confusing and uncommented, I'll do what I can to clarify them.  

Fwiw, the goal is to have the variable names be self explanatory.  There's too much mental model to keep in place if they don't work that way.  So I'm also very open to improvements we can make with the names itself.

Reply:
I was confused mostly by this particular key. Maybe we can call it `kopia-items-read`? or `stream-items-read`

Reply:
Sure, I'll wordsmith on it.